CHANGELOG.md

```diff
@@ -10,11 +10,13 @@
 * [UIBULKED-124](https://issues.folio.org/browse/UIBULKED-124) User-records - confirm changes - in app.
 * [UIBULKED-110](https://issues.folio.org/browse/UIBULKED-110) Populate holdings record identifiers dropdown
 * [UIBULKED-129](https://issues.folio.org/browse/UIBULKED-129) Preserve column selection from preview to Are you sure? form and confirmation page
+* [UIBULKED-115](https://issues.folio.org/browse/UIBULKED-115) In-app bulk edit form - holdings location.
 
 ## [2.0.3](https://github.com/folio-org/ui-bulk-edit/tree/v2.0.2) (2022-09-09)
 * [UIBULKED-144](https://issues.folio.org/browse/UIBULKED-144) Update behavior of uploading CSV file with updated records
 * [UIBULKED-141](https://issues.folio.org/browse/UIBULKED-141) Holdings record - preview of matched records
 
+
```

Comment:
👀

Reply:
@uladislausamets Thanks for noticing that, I missed it when was busy resolving the conflicts.